### ISSUE TITLE
Have you ever tested other backbonds?

### ISSUE THREAD

**Issue by goodluckcwl** [backtime92/CRAFT-Reimplementation#9]
Have you ever tested other backbonds, like ResNet, SENet?  The VGG16-BN is a structure proposed several years ago.

**Comment by backtime92** [comment]
@goodluckcwl No. In the [issues #34](https://github.com/clovaai/CRAFT-pytorch/issues/34#issuecomment-524525473) the author said the perforamnce difference was not significant with Resnet or PVAnet.
